subprojects/docs/src/docs/userguide/java_testing.adoc

```diff
@@ -300,49 +319,55 @@ The JUnit Platform serves as a foundation for launching testing frameworks on th
 
 The following code enables JUnit Platform support in `build.gradle`:
 
-++++
-<sample id="enablingJUnitPlatform" dir="testing/junitplatform/jupiter" title="Enabling JUnit Platform to run your tests">
-    <sourcefile file="build.gradle" snippet="enabling-junit-platform"/>
-</sample>
-++++
+=== Example: Enabling JUnit Platform to run your tests
 
-See api:org.gradle.api.tasks.testing.Test#useJUnitPlatform()[] for more details.
+[source.multi-language-sample,groovy]
+.build.gradle
+----
+include::{samplesPath}/testing/junitplatform/jupiter/build.gradle[tag=enabling-junit-platform]
+----
+
+See link:{javadocPath}/org/gradle/api/tasks/testing/Test.html#useJUnitPlatform-org.gradle.api.Action-[Test.useJUnitPlatform(org.gradle.api.Action)] for more details.
```

Comment:
The method reference changed: from `org.gradle.api.tasks.testing.Test#useJUnitPlatform()[]` to `useJUnitPlatform-org.gradle.api.Action-[Test.useJUnitPlatform(org.gradle.api.Action)`

Reply:
Oops. Fixed.